servers/partner-ci/9.x/vcenter.yaml

```diff
@@ -166,7 +166,18 @@
           testgroup_value_type: choices
           testgroup_value: '{obj:testgroup}'
 
-      - '{reporter-name}'
+      - '{reporter-name}':
```

Comment:
Reporter should use global variables, we don't need duplicate pieces of the code